public/about/team/index.html

```diff
@@ -198,6 +198,26 @@ <h5 class="card-title ">Patrick Sullivan</h5>
           </div>
         </div>
         <div class="card info-card bg-light col-lg-5 col-sm-12 mr-auto col-centered border-0 "></div>
```

Comment:
replace this line with \</div\> to format your card.

Reply:
![2021-08-20-113426_1197x515_scrot](https://user-images.githubusercontent.com/50334618/130257722-c3371fd7-f702-4ec6-8c9b-a5150db17048.png)